File: Web/RMA-Web-Shell/projects/fincare/src/app/payment-manager/models/commission-clawback-account-movement.ts
import { BaseClass } from 'projects/shared-models-lib/src/lib/common/base-class';
import { CommissionPaymentTypeEnum } from '../../shared/enum/commission-payment-type.enum';

export class CommissionClawBackAccountMovement extends BaseClass {
    clawBackAccountMovementId: number;
    clawBackAccountId: number;
    headerId: number;
    commissionPaymentType: CommissionPaymentTypeEnum;
    totalDueAmount: number;
    currentClawBackBalance: number;
    newClawBackBalance: number;
}

File: Service/FinCare/RMA.Service.FinCare.Contracts/Entities/Commissions/CommissionClawBackAccountMovement.cs
using RMA.Common.Entities;
using RMA.Service.Admin.MasterDataManager.Contracts.Enums;

namespace RMA.Service.FinCare.Contracts.Entities.Commissions
{
    public class CommissionClawBackAccountMovement : AuditDetails
    {
        public int ClawBackAccountMovementId { get; set; }
        public int ClawBackAccountId { get; set; }
        public int HeaderId { get; set; }
        public CommissionPaymentTypeEnum CommissionPaymentType { get; set; }
        public decimal TotalDueAmount { get; set; }
        public decimal CurrentClawBackBalance { get; set; }
        public decimal NewClawBackBalance { get; set; }
    }
}